File: 🟢 Easy/1903 - Largest Odd Number in String/_1903LargestOddNumberinString.js
/**
 * @param {string} num
 * @return {string}
 */
var largestOddNumber = function (num) {
    let i = num.length - 1;
    let to = 0;
    while (i >= 0) {
        if ((num.at(i) - '0') % 2 != 0) {
            to = i + 1;
            break;
        }
        i--;
    }

    return num.substring(0, to);
};

File: 🟢 Easy/1903 - Largest Odd Number in String/_1903LargestOddNumberinString.java
class Solution {
    public String largestOddNumber(String num) {
        int i = num.length() - 1;
        int to = 0;
        
        while (i >= 0) {
            if ((num.charAt(i) - '0') % 2 != 0) {
                to = i + 1;
                break;
            }
            i--;
        }

        return num.substring(0, to);
    }
}